### DISCUSSION TITLE
Unable to use GitHub Student Developer Pack benefits

### DISCUSSION THREAD

**Discussion by raicdev** [community/community#123243] [GitHub Education]
### Hi everyone,

One day, the GitHub Student Developer Pack suddenly became unavailable.
When I go to the application page, I am taken to the Pricing page.

**Comment by PingWang28** [comment]
Yes, the same . You need reverify you edu status. Check your mail box to find a mail from github to get the step and follow it.  I try it , but I always mislead to a wrong link finally .  --  https://github.com/pricing
